### ISSUE TITLE
Snapshots on fail

### ISSUE THREAD

**Issue by telangn** [damianszczepanik/cucumber-reporting#757]
I know my snapshots on fail are working because using another reporting program, the snapshots appear. But with this reporter, I am not seeing any snapshots in the report....Is it supported??

**Comment by damianszczepanik** [comment]
I don't understand what you are trying to say

**Comment by telangn** [comment]
Sorry, for not explaining better, let me try again. 

If a Cucumber scenario fails a test case while testing on the web browser, it will take a snapshot of the web browser screen using the following code: 

```
@After
	public void snapShotOnFail(Scenario scenario) throws Throwable {
		try {
			if (scenario.isFailed()) {
				final byte[] screenshot = ((TakesScreenshot) driver).getScreenshotAs(OutputType.BYTES);
				scenario.embed(screenshot, "image/png");
			}
		} catch (WebDriverException somePlatformsDontSupportScreenshots) {
			System.err.println(somePlatformsDontSupportScreenshots.getMessage());
		}
		cukelogs.info("Executed After Hook -- Snap shot on Failed Scenarios\r\n");
	}
```

The snapshot is saved in the cucumber json output file. After your  cucumber-reporting app produces the report, I do not see the snapshot in it for the failed test case. If I use another, following reporter app, I can see the snapshots - 
https://github.com/gkushang/cucumber-html-reporter

Does your app support this functionality. Because when my test case fails, I do not see a snapshot of the browser screen.

**Comment by telangn** [comment]
Snapshot means "screen capture"....a picture of the web browser @damianszczepanik

**Comment by damianszczepanik** [comment]
Show me JSON report

**Comment by telangn** [comment]
[cucumber.json.txt](https://github.com/damianszczepanik/cucumber-reporting/files/2351098/cucumber.json.txt)

@damianszczepanik  Please see attachment. Please remove the .txt extension so it is named cucumber.json only (Git would not accept .json as an attachment).
See lines 86-98, if you open it up in Sublime or something. Thanks!

**Comment by damianszczepanik** [comment]
"Packlink" snapshot is presented correctly

**Comment by telangn** [comment]
@damianszczepanik can you please explain your response. 

If you are saying that your program is working correctly is your snapshot presented in a specific HTML file. If so can you direct me to the correct one because I've looked through all reports and I cannot find it
